llvm/lib/Transforms/Vectorize/VPlanTransforms.cpp

```diff
@@ -935,6 +935,22 @@ static void simplifyRecipe(VPRecipeBase &R, VPTypeAnalysis &TypeInfo) {
 #endif
   }
 
+  VPValue *X;
+  VPValue *Y;
+  VPValue *X1;
+  VPValue *Y1;
+  // Simplify (X && Y) || (X1 && !Y1) -> X.
```

Comment:
```suggestion
  // Simplify (X && Y) || (X && !Y) -> X.
```
This now more closely matches the code, where repeated occurrences of X and Y are enumerated.

Reply:
Done, thanks!